app/src/main/java/org/glucosio/android/activity/AddGlucoseActivity.java

```diff
@@ -161,10 +165,9 @@ public void onNothingChosen(View labelledSpinner, AdapterView<?> adapterView) {
             p = getIntent().getExtras();
             reading = p.getString("reading");
             if (reading != null) {
-                // If yes, first convert the decimal value from Freestyle to Integer
+                // If yes, first convert the decimal value from Freestyle to Double
```

Comment:
Unnecessary comment, the code explain itself.

Reply:
Removed!